src/release.js

```diff
@@ -0,0 +1,62 @@
+const fs = require('fs');
+const util = require('util');
+
+const readFile = util.promisify(fs.readFile);
+const exists = util.promisify(fs.exists);
+
+const {validateChangelog} = require('./validate');
+
+const validateRelease = async (changelog, branch) => {
+    console.info(changelog);
+    const folder = changelog.replace('CHANGELOG.md', '');
+
+    const changelogContent = await readFile(changelog, {encoding: 'utf-8'});
+
+    const {
+        isUnreleased,
+        version,
+        date,
+        skeleton,
+    } = validateChangelog(changelogContent);
+
+    if (isUnreleased) {
+        throw new Error(`"${branch}" branch can't be unreleased`);
```

Comment:
here and in other cases better use project name from package json instead of branch

Reply:
what is unreleased?

Reply:
The idea is to show something like `"release/app" branch can't be unreleased`, maybe is better if it simply says `A release branch can't have "Unreleased" version`

Reply:
yea, sounds better